velox/exec/tests/TableWriteTest.cpp

```diff
@@ -660,103 +658,147 @@ class TableWriteTest : public HiveConnectorTestBase {
 
 class PartitionedTableWriterTest
     : public TableWriteTest,
-      public testing::WithParamInterface<uint32_t> {
+      public testing::WithParamInterface<uint64_t> {
  public:
   PartitionedTableWriterTest() : TableWriteTest(GetParam()) {}
 
-  static std::vector<uint32_t> getTestParams() {
-    std::vector<uint32_t> testParams;
+  static std::vector<uint64_t> getTestParams() {
+    std::vector<uint64_t> testParams;
     testParams.push_back(TestParam{
+        FileFormat::DWRF,
         TestMode::kPartitioned,
         CommitStrategy::kNoCommit,
         HiveBucketProperty::Kind::kHiveCompatible,
         false}
                              .value);
     testParams.push_back(TestParam{
+        FileFormat::DWRF,
         TestMode::kPartitioned,
         CommitStrategy::kTaskCommit,
         HiveBucketProperty::Kind::kHiveCompatible,
         false}
                              .value);
     testParams.push_back(TestParam{
+        FileFormat::DWRF,
         TestMode::kBucketed,
         CommitStrategy::kNoCommit,
         HiveBucketProperty::Kind::kHiveCompatible,
         false}
                              .value);
     testParams.push_back(TestParam{
+        FileFormat::DWRF,
         TestMode::kBucketed,
         CommitStrategy::kTaskCommit,
         HiveBucketProperty::Kind::kHiveCompatible,
         false}
                              .value);
     testParams.push_back(TestParam{
+        FileFormat::DWRF,
         TestMode::kBucketed,
         CommitStrategy::kNoCommit,
         HiveBucketProperty::Kind::kPrestoNative,
         false}
                              .value);
     testParams.push_back(TestParam{
+        FileFormat::DWRF,
         TestMode::kBucketed,
         CommitStrategy::kTaskCommit,
         HiveBucketProperty::Kind::kPrestoNative,
         false}
                              .value);
+    if (hasWriterFactory(FileFormat::PARQUET)) {
```

Comment:
Can we also cover bucketed table case?

Reply:
There is an issue with partitioned and bucketed cases
See https://github.com/facebookincubator/velox/issues/5560
I will fix and add them.